Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -28,11 +28,11 @@
 #include "VideoBackends/D3D12/VideoBackend.h"
 #endif
 #include "VideoBackends/Null/VideoBackend.h"
-#ifdef HAS_OPENGL
+#if defined(HAS_OPENGL) && !_UWP
```

Comment:
It would likely be preferable to do this. Preferably one should just not define `HAS_OPENGL` to on UWP, which is [conditionally defined here](https://github.com/dolphin-emu/dolphin/blob/1fd8d476e84b9362648d65f67f833719bad5673b/Source/VSProps/Base.Dolphin.props#L42). Ditto for `HAS_VULKAN`.